src/helm.exec.ts

```diff
@@ -274,6 +274,31 @@ export function helmGet(resourceNode?: ClusterExplorerNode) {
     });
 }
 
+export function helmUninstall(resourceNode?: ClusterExplorerNode) {
+    if (!resourceNode) {
+        return;
+    }
+    if (resourceNode.nodeType !== NODE_TYPES.helm.release) {
+        return;
+    }
+    const releaseName = resourceNode.releaseName;
+    logger.log("⎈⎈⎈ Uninstalling " + releaseName);
+    vscode.window.showWarningMessage(`This will uninstall ${releaseName}, this cannot be undone.`, 'Uninstall').then((opt) => {
```

Comment:
I would change the message... maybe `You are about to uninstall ${releaseName}. This action cannot be undone.`

Reply:
Thank you, improved the message.